sdk/include/reactos/debug/rappdbg.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * PROJECT:     ReactOS headers
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Debugging the ReactOS applications
+ * COPYRIGHT:   Copyright 2021 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#pragma once
+
+#ifdef NDEBUG
+#undef DBG
+#endif
+
+#ifndef _INC_WINDOWS
+#include <windows.h>
+#endif
+
+#if !defined(_STRSAFE_H_INCLUDED_) && !defined(NO_STRSAFE)
+#include <strsafe.h>
+#endif
+
+#include <assert.h>
+
+#ifndef ASSERT
+#define ASSERT assert
+#endif
+
+#ifndef APPDBG_BUFSIZE
+#define APPDBG_BUFSIZE 512
+#endif
+
+#ifndef DPRINT
+# if DBG
+
+static inline ULONG rappdbg_printfA(const char *file, int line, const char *fmt, ...)
+{
+    char buf[APPDBG_BUFSIZE];
+    va_list va;
+    const char *fname;
+    size_t cch;
+
+    va_start(va, fmt);
+
+    fname = strrchr(file, '\\');
```

Comment:
Why not keeping the original file path as it comes out? (this is what the regular DPRINT etc. macros do)

Reply:
Fixed.